### ISSUE TITLE
[BUG]: System_log = False not possible in Setup

### ISSUE THREAD

**Issue by jnsflt** [pycaret/pycaret#2718]
### pycaret version checks

- [X] I have checked that this issue has not already been reported [here](https://github.com/pycaret/pycaret/issues).

- [X] I have confirmed this bug exists on the [latest version](https://github.com/pycaret/pycaret/releases) of pycaret.

- [ ] I have confirmed this bug exists on the develop branch of pycaret (pip install -U git+https://github.com/pycaret/pycaret.git@develop).

### Issue Description

Hi!

Due to the issue of log files creating 400 MB of WARNING lines every run, I tried to set the parameter "system_log" in setup to False. This caused an error, which I believe is unintentional. On github, the parameter "system_log" has the following description:

>         system_log: bool or str or logging.Logger, default = True
>             Whether to save the system logging file (as logs.log). If the input
>             is a string, use that as the path to the logging file. If the input
>             already is a logger object, use that one instead.

So it should be possible to set system_log to False.

### Reproducible Example

```python
from pycaret.datasets import get_data
exp = ClassificationExperiment()
df = get_data("titanic")

exp.setup(data=df, target="Survived", system_log=False)
```

### Expected Behavior

Setup proceeding as usual but without any logs.log file created.

### Actual Results

```python-traceback
File "...lib/python3.8/site-packages/pycaret/internal/logging.py", line 107, in create_logger
    return DummyLogger()
TypeError: __init__() missing 1 required positional argument: 'name'
```

### Installed Versions

<details>
3.0.0rc2</details>

**Comment by tvdboom** [comment]
this issue has already been fixed in pycaret 3.0. Install the pre-release to avoid it.

**Comment by jnsflt** [comment]
I am using pycaret 3.0.0rc2?
